emulator/include/Emulator.hpp

```diff
@@ -46,6 +54,75 @@ class OswEmulator {
     CPUState cpustate = CPUState::deepSleep;
     bool autoWakeUp = true;
     bool wakeUpNow = false;
+
+    // Web-interface
+
+    // CheckBox section
+    bool checkBoxTimeFormat = false;
+    bool checkBoxSettingDisplayOverlays = false;
+    bool checkBoxSettingDisplayOverlaysOnWatchScreen = false;
+    bool checkBoxRaiseToWakeEnabled = false;
+    bool checkBoxButtonToWakeEnabled = false;
+    bool checkBoxTapToWakeEnabled = false;
+    bool checkBoxLightSleepEnabled = false;
+    bool checkBoxStepsHistoryClear = false;
+
+    // Color section
+    float colorThemeBackgroundColor[3] = {0/ 255.0F, 0/ 255.0F, 0/ 255.0F};
+    float colorThemeBackgroundDimmedColor[3] = {64/ 255.0F, 64/ 255.0F, 64/ 255.0F};
+    float colorThemeForegroundColor[3] = {255/ 255.0F, 255/ 255.0F, 255/ 255.0F};
+    float colorThemeForegroundDimmedColor[3] = {122/ 255.0F, 122/ 255.0F, 122/ 255.0F};
+    float colorThemePrimaryColor[3] = {0/ 255.0F, 209/ 255.0F, 178/ 255.0F};
+    float colorThemeInfoColor[3] = {32/ 255.0F, 156/ 255.0F, 238/ 255.0F};
+    float colorThemeSuccessColor[3] = {72/ 255.0F, 199/ 255.0F, 116/ 255.0F};
+    float colorThemeWarningColor[3] = {255/ 255.0F, 221/ 255.0F, 87/ 255.0F};
+    float colorThemeDangerColor[3] = {255 / 255.0F, 56 / 255.0F, 96 / 255.0F};
+
+    // Dropdown section
+    // const char* itemsDateFormat[] = {"mm/dd/yyyy","dd.mm.yyyy"};
+    int comboDateFormat = 0;
+
+    // const char* itemsSettingDisplayDefaultWatchface[] = {"analog","digital","binary"};
+    int comboSettingDisplayDefaultWatchface = 0;
+
+    // input editor (int) section
+    int inputintSettingDisplayBrightness = DISPLAY_BRIGHTNESS;
+    int inputintSettingDisplayTimeout = DISPLAY_TIMEOUT;
+    int inputintRaiseToWakeSensitivity = WAKE_FROM_RAISE_SENSITIVITY;
+    int inputintTimeZone = CONFIG_TIMEZONE;
+    int inputintStepsPerDay = STEPS_PER_DAY;
+
+    // input editor (float) section
+    float inputfloatDaylightOffset = CONFIG_DAYLIGHTOFFSET;
+
+    void* keyList[25] = {
```

Comment:
This makes extension pretty hard... Why?

Reply:
It draws an overlay, which shows a display border... We should do that in SDL directly - as this is a job of the emulator.